services/traction/api/endpoints/routes/v1/tenant/issuer/credentials.py

```diff
@@ -0,0 +1,141 @@
+import logging
+from typing import List
+
+from fastapi import APIRouter, Depends
+from sqlalchemy.ext.asyncio import AsyncSession
+from starlette import status
+
+from api.services.v1 import issuer_service
+from api.services.v1.issuer_service import IssueCredentialData
+
+from api.db.models.v1.contact import Contact
+
+from api.endpoints.dependencies.tenant_security import get_from_context
+from api.endpoints.dependencies.db import get_db
+
+
+from api.endpoints.models.credentials import (
+    IssueCredentialProtocolType,
+    CredentialPreview,
+)
+from api.endpoints.models.tenant_workflow import (
+    TenantWorkflowStateType,
+)
+
+from api.endpoints.models.v1.issuer import CredentialsListResponse, CredentialItem
+
+router = APIRouter()
+logger = logging.getLogger(__name__)
+
+
+@router.get("/credentials", response_model=CredentialsListResponse)
+async def get_issued_credentials(
+    state: TenantWorkflowStateType | None = None,
+    workflow_id: str | None = None,
```

Comment:
these workflow ids are internal, we aren't returning any internal data through the API (except where the id is actually used by LOB - ex. schema_id) or through the acapy data field that is explicitly asked for.

Reply:
removed that as an option for the v1 endpoint, good catch.